app/client/src/sagas/__tests__/onPageUnloadSaga.test.ts

```
@@ -0,0 +1,394 @@
+                  },
+                ],
+              },
+            },
+          ]);
+        });
+
+      // Replace the mocked function temporarily
+      const originalHandleExecuteJSFunctionSaga = handleExecuteJSFunctionSaga;
+
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      (handleExecuteJSFunctionSaga as any) = mockHandleExecuteJSFunctionSaga;
```

Biome (1.9.4)

[error] The imported variable handleExecuteJSFunctionSaga is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)